services/traefik/docker-compose.aws.yml

```diff
@@ -31,6 +31,11 @@ services:
       - "--entryPoints.https.forwardedHeaders.insecure"
       - "--providers.file.directory=/etc/traefik/"
       - "--providers.file.watch=true"
+      - '--entryPoints.redis.address=:6379'
+    ports:
+      - target: 6379
+        published: 6379
+        mode: host
```

Comment:
Q: why you use mode host here? the port will not be accessible from any machine in the swarm then.

Reply:
This is how we expose ports on traefik. The idea is to only expect traffic on nodes where we run traefik